src/rest/Endpoints.ts

```diff
@@ -270,4 +270,36 @@ export default class Endpoints {
   public static GUILD_VANITY_URL(guildID: string) {
     return `/guilds/${guildID}/vanity-url`;
   }
+
+  /**
+   * `/users/@me`
+   * - GET
+   * - PATCH
+   */
+  public static CURRENT_USER() {
```

Comment:
for consistency would rename this SELD instead of CURRENT
```suggestion
  public static SELF_USER() {
```